runchecks.sh

```diff
@@ -16,15 +20,10 @@ echo "Running isort"
 isort --check-only --diff --quiet \
  || { echo "isort errors found! Run 'isort' with no options to fix."; exit 1; }
 
-echo "Running shellcheck"
-git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck
+echo "Running shellcheck (if availablein PATH)"
+if hash shellcheck 2>/dev/null; then
+  git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck
+fi
 
 echo "Running test docs generation"
 mkdocs build
-
-echo "Running Django system checks"
-# See .travis.yml for explanation of the environment variable overriding.
-SITE_URL="https://treeherder.dev" TREEHERDER_DEBUG="False" python -bb ./manage.py check --deploy --fail-level WARNING
-
-echo "Running Python tests"
-pytest --cov --cov-report=xml tests/
```

Comment:
How do we run tests inside the container?  The tests run inside the docker container is different than the tests running outside the docker container.  We need both, or else there is going to be a lot more pushes to find out how the linux version fails

Reply:
It is documented down here how to run tests inside the container:
https://github.com/mozilla/treeherder/pull/5943/files#diff-5a61e9f9f0b3221e1b12f6e24cf94ed4R77

The tests do fail the same way for Mac and Linux because we're not doing platform specific tests. In the case of Windows we do skip some tests.

People can still run tests within Docker if they prefer to, however, for integration with IDEs it is more functional the non-Docker set up.

Reply:
Fine, I will make another PR after this one lands with the extra instructions for running tests in Docker too.